template/module/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © <YEAR(s)> <AUTHOR(s)>
+
+# Instructions: https://www.odoo.com/documentation/8.0/reference/module.html
+{
+    "name": "Module name",
+    "summary": "Module summary",
+    "version": "8.0.1.0.0",
+    "category": "Module category",
+    "license": "AGPL-3",
+    "website": "https://odoo-community.org/",
+    "author": "Odoo Community Association (OCA)",
```

Comment:
Add a placeholder for the actual author name: 

    "author": "{author name}, Odoo Community Association (OCA)"